java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -1,6 +1,8 @@
 /** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
 package glide.api.commands;
 
+import glide.api.models.commands.RangeOptions.IRangeQuery;
+import glide.api.models.commands.RangeOptions.IScoredRangeQuery;
```

Comment:
Let's split IRangeQuery and IScoredRangeQuery into separate files.  
```suggestion
import glide.api.models.commands.ScoredRangeOptions.IScoredRangeQuery;
```